packages/vscode-docker-registries/src/contracts/RegistryDataProvider.ts

```diff
@@ -16,5 +16,5 @@ export interface RegistryDataProvider<T extends RegistryItem> extends vscode.Tre
     getLoginInformation?(item: T): Promise<LoginInformation> | LoginInformation;
 
     onConnect?(): Promise<void>;
-    onDisconnect?(): Promise<void>;
+    onDisconnect?(item?: T): Promise<void>;
```

Comment:
I don't think adding this `item?: T` will be needed with the above changes to the connect/disconnect tracked registry commands

Reply:
yup you're right